src/shared-components/CRUDModalTemplate/CRUDModalTemplate.spec.tsx

```
@@ -428,51 +433,53 @@ describe('CreateModal', () => {
           onClose={mockOnClose}
           onSubmit={mockOnSubmit}
         >
-          <Form.Group>
-            <Form.Label>Name</Form.Label>
-            <Form.Control type="text" placeholder="Enter name" />
-          </Form.Group>
+          <div>
+            <label>Name</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)